<?xml version="1.0"?>
<launch>

    <node pkg="motion_detection" type="motion_detection" name="motion_detection" output="screen" respawn="false">
        <remap from="~input_image" to="/tower_cam3d/rgb/image_raw"/>        
        <remap from="~input_pointcloud" to="/tower_cam3d/depth_registered/points"/>        
        <remap from="~input_camerainfo" to="/tower_cam3d/rgb/camera_info"/>        

        <param name="x_trans" type="double" value="0.05" />
        <param name="y_trans" type="double" value="0.0" />
        <param name="z_trans" type="double" value="0.0" />
        <param name="roll"    type="double" value="0.0" />
        <param name="pitch"   type="double" value="0.0" />
        <param name="yaw"     type="double" value="0.0" />
    </node>

    <node name="expected_flow_image" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/motion_detection/expected_flow_image"/>
    </node>

    <node name="optical_flow_image" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/motion_detection/optical_flow_image"/>
    </node>
</launch>
